### ISSUE TITLE
.gitignore cannot be part of starter

### ISSUE THREAD

**Issue by s-spindler** [helm/helm#4059]
If the `.helmignore` file in a starter contains a line with `.gitignore` (as is the case for the default starter) the `.gitignore` file will be missing from the created chart directory.
This means you will either have to accept not having a `.gitignore` file in your starter or remove the line from `.helmignore` which would mean always having to add it again manually or having helm package it.

Is helm taking the contents of a starter's `.helmignore` file into account when creating a chart directory from it?

Helm version: 2.9.0

**Comment by s-spindler** [comment]
@bacongobbler I think I should not have put this as a question; let me rephrase:

Helm should not take the contents of a starter's `.helmignore` file into account when creating a chart directory from it.

I do consider this a bug in helm - what is your opinion on this? Would you be open to a PR for this?

**Comment by bacongobbler** [comment]
sure.

**Comment by bacongobbler** [comment]
closing as inactive.
